File: golangs/call_sum.go
package main

import "fmt"

func sum2(bil1, bil2 int) int {
	return (bil1 + bil2)
}

func main() {
	var b = [5]int{10, 5, 6, -7, -8}
	var r int = 0
	r = sum2(b[0], b[1])
	fmt.Println(r)

	r = sum2(r, b[2])
	fmt.Println(r)

	r = sum2(r, b[3])
	fmt.Println(r)

	r = sum2(r, b[4])
	fmt.Println(r)

}

File: luas/call_sum.lua
function sum2(bil1, bil2)
    return(bil1 + bil2)
end 

b = {10, 5, 6, -7, -8}

r = sum2(b[1], b[2])
print(r)

r = sum2(r, b[3])
print(r)


r = sum2(r, b[4])
print(r)


r = sum2(r, b[5])
print(r)
